### PR TITLE
AI-Goat added

### PR DESCRIPTION
**Pull request by mrtlgz**
A new vulnerable AI application included

### CODE REVIEW

File: src/data/collection.json [hunk 1]
```diff
@@ -32,6 +32,25 @@
...
+		],
+		"references": [
+			{
+				"name": "download",
+				"url": "https://github.com/dhammon/ai-goat"
+			}
+		],
+		"author": "IBM/Watchfire",
+		"notes": "Log in with jsmith/demo1234 or admin/admin",
+		"badge": "hclproducts/AltoroJ"
```
Comment (kingthorin) [inline src/data/collection.json:52]: This doesn't seem right

File: src/data/collection.json [hunk 2]
```diff
@@ -32,6 +32,25 @@
...
 		"badge": null
 	},
+	{
+		"url": "https://github.com/dhammon/ai-goat",
+		"name": "AI-Goat",
+		"collection": [
+			"offline"
+		],
+		"technology": [
+			"Python"
```
Comment (kingthorin) [inline src/data/collection.json:42]: Add something about the LLM in use?

File: src/data/collection.json (hunk 1 above)
Comment (mrtlgz) [reply src/data/collection.json:52]: I've forked the master branch, so it was likely there already. I'm not entirely sure,.

File: src/data/collection.json (hunk 2 above)
Comment (mrtlgz) [reply src/data/collection.json]: Done!

File: src/data/collection.json (hunk 1 above)
Comment (mrtlgz) [reply src/data/collection.json]: Yeah noticed it, sorry for the trouble.

**Pull request merged**